build.gradle

```diff
@@ -278,6 +282,22 @@ project(':iceberg-mr') {
       exclude group: 'org.apache.avro', module: 'avro'
     }
 
+    testCompile("com.klarna:hiverunner:4.1.0") {
```

Comment:
I can remove this if we want to merge it into the PR on Iceberg

Reply:
I think we need to have something here that demonstrates the problem. Let's remove it from this PR and we can then merge everything else in to the branch so it shows up in the Iceberg PR we have open. Can you then please raise a separate PR like this one here against our branch which has this and then also a simple HiveRunner test. Can we reproduce the issue by just doing an example HiveRunner test which creates a table and queries it? i.e. without using the InputFormat at all? That would be a nice simple way to demonstrate the issue at least without having to merge in a whole lot of other code.